CHANGES.md

```diff
@@ -1,5 +1,10 @@
 # Change Log
 
+## 2.2.1
+
+- Refactor params for Oneshot search
+- Change ResultReader to JSONResultReader
```

Comment:
Worth clarifying the Changelog what this change helped to solve
```suggestion
- Change ResultReader to JSONResultReader to fix ...
```

Reply:
updated the reason!